endpoints/authentication/src/main.cpp

```diff
@@ -58,37 +59,16 @@ namespace irods::http::handler
 		net::ip::tcp::resolver tcp_res{io_ctx};
 		beast::tcp_stream tcp_stream{io_ctx};
 
-		// Setup curl
-		CURLU* endpoint{curl_url()};
-		if (endpoint == nullptr) {
-			std::abort();
-		}
-
-		// Parse url
-		CURLUcode rc{curl_url_set(endpoint, CURLUPART_URL, token_endpoint.c_str(), 0)};
-		if (rc != 0) {
-			log::debug("Something happend....");
-		}
-
-		// Get host
-		char* host{};
-		rc = curl_url_get(endpoint, CURLUPART_HOST, &host, 0);
-		if (rc != 0) {
-			log::debug("Something happend....");
-		}
+		auto parsed_uri{boost::urls::parse_uri(token_endpoint)};
+		auto url{*parsed_uri};
```

Comment:
We should probably still check the `parsed_uri`, just not sure how we want to 'fail' here...

Reply:
Checked.